app/log-trace-message.js

```diff
@@ -0,0 +1,7 @@
+const appInsights = require('applicationinsights')
+
+module.exports = (message) => {
+  if (appInsights.defaultClient !== undefined) {
```

Comment:
Maybe add another method like `trackException` so easier for user to add in other methods.

Reply:
Added.  I've also exposed the app insights capability through the npm API so teams can use it in their consuming services.